tiatoolbox/tools/registration/wsi_registration.py

```diff
@@ -172,3 +176,259 @@ def match_histograms(image_a, image_b, kernel_size=7):
         image_a = exposure.match_histograms(image_a, image_b).astype(np.uint8)
 
     return image_a, image_b
+
+
+class DFBRegistration:
+    r"""Deep Feature based Registration
+
+    This class implements a CNN feature based registration,
+    as proposed in a paper titled `Deep Feature based Cross-slide Registration
+    <https://arxiv.org/pdf/2202.09971.pdf>`_.
+
+    """
+
+    def __init__(self):
+        self.patch_size = (224, 224)
```

Comment:
```suggestion
        self.patch_size: Tuple[int] = (224, 224)
```

Use typehints as here https://github.com/TissueImageAnalytics/tiatoolbox/blob/4dcf9d765d82e7918ff026fd60b48af65cbb89da/tiatoolbox/models/architecture/micronet.py#L287

Reply:
https://github.com/TissueImageAnalytics/tiatoolbox/blob/1696b7a5b86df6d7fa1ddf5b5c1c68dd319f7847/tiatoolbox/tools/registration/wsi_registration.py#L290